src/coreclr/src/jit/emit.cpp

```diff
@@ -1211,8 +1211,13 @@ void emitter::appendToCurIG(instrDesc* id)
 
 #ifdef DEBUG
 
-void emitter::emitDispInsOffs(unsigned offs, bool doffs)
+void emitter::emitDispInsOffs(unsigned offs, bool doffs, BYTE* code)
```

Comment:
This is a convenient place to put this code, but it muddies the "display instruction offset" functionality.

Why not just create a new `emitDispInsAddr` function and call it in the 3 per-architecture places where `emitDispInsOffs` is currently called?